docker/docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+stgrest:
+  image: begriffs/postgrest:latest
```

Comment:
At this stage this refers to 0.3.2.1 on docker hub. To test this docker-compose you'll need to build an image locally and refer to that until we tag this repo with `v0.4.0.0-1` and trigger a docker hub build.